### PR TITLE
[extension](kettle) add kettle plugins to write doris

### PR DESCRIPTION
**Pull request by JNSimba**
## Proposed changes
 

add kettle plugins to write doris
Implementation principle: 
1. Receive data from upstream
2. Serialize it into data that can be written by Doris
3. Cache this data in the memory buffer
4. After reaching the threshold of Maximum rows/bytes, write it to Doris through Stream Load.

### CODE REVIEW

**Comment by JNSimba** [comment]
run buildall

**Comment by JNSimba** [comment]
run buildall

**Comment by JNSimba** [comment]
run buildall

**Comment by JNSimba** [comment]
run buildall

File: extension/kettle/ui/src/main/resources/org/pentaho/di/ui/trans/steps/dorisstreamloader/messages/messages_en_US.properties [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+DorisStreamLoaderDialog.Shell.Title=Doris Stream Loader
+DorisStreamLoaderDialog.Stepname.Label=Step name
+DorisStreamLoaderDialog.Fenodes.Label=Fenodes
+DorisStreamLoaderDialog.DatabaseName.Label=DataBase
```
Comment (morningman) [inline extension/kettle/ui/src/main/resources/org/pentaho/di/ui/trans/steps/dorisstreamloader/messages/messages_en_US.properties:22]:
```suggestion
DorisStreamLoaderDialog.DatabaseName.Label=Database
```

**Comment by JNSimba** [comment]
run buildall

**Review by morningman — APPROVED** [review]

**Review by CalvinKirs — APPROVED** [review]

**Pull request merged**